module/VuFind/src/VuFind/AjaxHandler/PluginManager.php

```diff
@@ -51,7 +51,6 @@ class PluginManager extends \VuFind\ServiceManager\AbstractPluginManager
         'deleteRecordComment' => DeleteRecordComment::class,
         'doiLookup' => DoiLookup::class,
         'getACSuggestions' => GetACSuggestions::class,
-        'getFacetData' => GetFacetData::class,
```

Comment:
I see that the factory has been deleted, but not the class itself. Is this intentional?